schemas/testPlanSchema.js

```diff
@@ -4,7 +4,7 @@ export const testPlanSchema = yup.object().shape({
   _id: yup.string().optional(),
   name: yup.string().required(),
   description: yup.string().required(),
-  customerFeedback: yup.string().optional(),
+  customerFeedback: yup.string().required(),
```

Comment:
This is for the later on iteration process in which customer can provide feedback. However, when an engineer first create a test plan, most likely there won't be any feedback. This should definitely be optional.

Reply:
feedback is due for a major refactor anyways. Ideally it would be just a feedback field. This field would consist of an array of objects containing users and comments.

Reply:
Will temporarily change back to optional, but needs reworking later.